src/System.Net.NameResolution/src/System.Net.NameResolution.csproj

```diff
@@ -10,6 +10,9 @@
     <OutputType>Library</OutputType>
```

Comment:
Yes we should clean that up. I have a script that I will be running again soon that will catch any new ones that have happen since I did i last month.